app/include/dt-bindings/zmk/matrix-transform.h

```diff
@@ -4,7 +4,4 @@
  * SPDX-License-Identifier: MIT
  */
 
-#define KT_ROW(item) (item >> 8)
-#define KT_COL(item) (item & 0xFF)
-
-#define RC(row, col) (((row) << 8) + (col))
\ No newline at end of file
+#include "matrix_transform.h"
```

Comment:
Ditto

Reply:
`#warning "'matrix-transform.h' has been deprecated and replaced by 'matrix_transform.h'"`

Reply:
Ditto above.